it/src/test/java/org/corfudb/universe/scenario/WriteAfterResetIT.java

```diff
@@ -118,7 +118,13 @@ private void writeAfterReset(UniverseWorkflow<Fixture<UniverseParams>> wf) throw
                 }
             }).exceptionally(ex -> {
                 // Bootstrap a new 1 node cluster.
-                corfuCluster.bootstrap();
+                try {
+                    corfuCluster.bootstrap();
+                }
+                catch (Exception e) {
+                    throw new RuntimeException(e);
```

Comment:
Add a long information here. So we know where the exception is from

Reply:
Sounds good